source/documentation/continuous-integration-and-delivery/example-pipelines.md

```diff
@@ -0,0 +1,68 @@
+## Example pipelines
+
+### Deploy an application to GOV.UK PaaS
+
+This pipeline clones a public open source git repository, and deploys it to GOV.UK PaaS
+
+#### Requirements
+
+There must be the following variables set:
+
+- `paas-username`
+- `paas-password`
+
+#### Pipeline
+
+<pre><code>---
```

Comment:
Out of interest, is that better than:

~~~markdown
```yaml
---
name: pipeline
```
~~~

Reply:
It renders weirdly with nested codeblocks due to the indentation; this may be a bug